readme.md

```diff
@@ -350,6 +350,20 @@ queryString.parseUrl('https://foo.bar?foo=bar');
 //=> {url: 'https://foo.bar', query: {foo: 'bar'}}
 ```
 
+##### parseFragmentIdentifier
```

Comment:
Moving this under the `parseUrl` section as this option does not apply to parse

Reply:
You need to make it clearer that it's an option.

And https://github.com/sindresorhus/query-string/pull/222/commits/66379d5ede6e8301233096c6f85a8bcb69b0e5cf#diff-0730bb7c2e8f9ea2438b52e419dd86c9R342 is still not correct.